source/plugins/projects/projects/index.js

```diff
@@ -1023,7 +1023,7 @@ let projects = {
 	 * 
 	 * @example
 	 * 
-	 * deleteFolder('MyNewProject', 'C:\Users\User\Desktop\folder');
+	 * deleteFolder(project, 'C:\Users\User\Desktop\folder');
```

Comment:
The second parameter is a folder within the project, it cannot be with a full path, it has to start with /...

Reply:
The path still starts with `C:\...`

Reply:
Same as deleteFile